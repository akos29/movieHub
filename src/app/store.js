import { configureStore } from '@reduxjs/toolkit';
import movieReducer from '../features/movie/movieSlice';

const store = configureStore({
  reducer: {
    movies: movieReducer,
  },
});

export default store;
