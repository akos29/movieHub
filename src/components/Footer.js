import React from 'react';

function Footer() {
  return (
    <footer>
      All right reserved.
    </footer>
  );
}

export default Footer;
