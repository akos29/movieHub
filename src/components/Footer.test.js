import React from 'react';
import TestRenderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../app/store';
import Footer from './Footer';

it('Footer Component snapshot test', () => {
  const footer = TestRenderer
    .create(
      <Provider store={store}>
        <Footer />
      </Provider>,
    )
    .toJSON();
  expect(footer).toMatchSnapshot();
});
