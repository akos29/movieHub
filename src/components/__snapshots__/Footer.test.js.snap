// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component snapshot test 1`] = `
<footer>
  All right reserved.
</footer>
`;
